applications/asset_tracker_v2/src/ext_sensors/ext_sensors_bsec.c

```diff
@@ -26,21 +30,46 @@ LOG_MODULE_REGISTER(ext_sensors_bsec, CONFIG_EXTERNAL_SENSORS_LOG_LEVEL);
  */
 #if defined(CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_ULTRA_LOW_POWER)
 #define BSEC_SAMPLE_RATE BSEC_SAMPLE_RATE_ULP
+#define BSEC_SAMPLE_PERIOD_S 300
 #elif defined(CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_LOW_POWER)
 #define BSEC_SAMPLE_RATE BSEC_SAMPLE_RATE_LP
+#define BSEC_SAMPLE_PERIOD_S 3
 #elif defined(CONFIG_EXTERNAL_SENSORS_BSEC_SAMPLE_MODE_CONTINUOUS)
-#define BSEC_SAMPLE_RATE BSEC_SAMPLE_RATE_HP
+#define BSEC_SAMPLE_RATE BSEC_SAMPLE_RATE_CONT
+#define BSEC_SAMPLE_PERIOD_S 1
 #endif
 
 /* Temperature offset due to external heat sources. */
 float temp_offset = (CONFIG_EXTERNAL_SENSORS_BSEC_TEMPERATURE_OFFSET / (float)100);
 
+static const bsec_sensor_configuration_t requested_virtual_sensors[4] = {
```

Comment:
A comment on what this variable is used for would be helpful.

Reply:
done